src/java.base/share/classes/sun/util/calendar/ZoneInfoFile.java

```diff
@@ -275,21 +232,13 @@ public Void run() {
     }
 
     private static void addOldMapping() {
-        for (String[] alias : oldMappings) {
-            aliases.put(alias[0], alias[1]);
-        }
-        if (USE_OLDMAPPING) {
-            aliases.put("EST", "America/New_York");
-            aliases.put("MST", "America/Denver");
-        } else {
-            aliases.put("EST", "America/Panama");
-            aliases.put("MST", "America/Phoenix");
+        for (var key : ZoneId.SHORT_IDS.keySet()) {
+            switch (key) {
+                case "EST" -> aliases.put("EST", "America/Panama");
+                case "MST" -> aliases.put("MST", "America/Phoenix");
+                default -> aliases.put(key, ZoneId.SHORT_IDS.get(key));
```

Comment:
"EST" and "MST" case are no longer needed? ZoneId should already map them to "America/Panama" and "America/Phoenix". We no longer offer the old mapping, so we can safely follow ZoneId.SHORT_IDS.

Reply:
Right. Removed them.